### PR TITLE
Klab simulation

### PR DESCRIPTION
**Pull request by oysand**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -83,12 +83,30 @@ http://localhost:5000/stream_viewer?topic=/camera/rgb/image_raw
...
+
 
 ### Running custom model
 
-To run the simulation with the custom model, set `WORLD_NAME=new_world` in `docker_scripts/entrypoint.sh`.
+To run the simulation with the custom model set the WORLD_NAME as an argument or environment variable like this:
 
+```bash
+WORLD_NAME=your_custom_model docker-compose up --build
+```
```
Comment (GodVenn) [inline README.md:109]: Docker compose automatically reads environment variables from a .env file in the root folder where the command is run from, could we use this instead?

File: worlds/house.world [hunk 2]
```diff
@@ -11,7 +11,7 @@
     </include>
 
     <physics type="ode">
-      <real_time_update_rate>1000.0</real_time_update_rate>
+      <real_time_update_rate>0000.0</real_time_update_rate>
```
Comment (GodVenn) [inline worlds/house.world:14]: Is this supposed to be here?

**Review by GodVenn — APPROVED** [review]
Really nice, great work!
Just left a thought on how to parse environment variables

File: worlds/house.world (hunk 2 above)
Comment (oysand) [reply worlds/house.world:14]: That one is just my personal preference, got added by mistake

File: README.md (hunk 1 above)
Comment (oysand) [reply README.md]: That could definitely be an option. Tried to change the text to highlight this option clearer, but kept the argument option as an example as both option works.

**Pull request merged**